basictracer/binary_propagator.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+
+import struct
+from .context import SpanContext
+from .propagator import Propagator
+# This can cause problems when old versions of protobuf are installed
+from .wire_pb2 import TracerState
+from opentracing import InvalidCarrierException
+
+_proto_size_bytes = 4  # bytes
+
+
+class BinaryPropagator(Propagator):
```

Comment:
the point of this change is that a BasicTracer extender may choose not to even import this BinaryPropagator file and its `.wire_pb2` dependency that breaks some builds that also involve other version of protobufs.